### ISSUE TITLE
Deployment indicates successful but MDS stuck in creating on new build

### ISSUE THREAD

**Issue by jeevadotnet** [ceph/ceph-ansible#6007]
**Bug Report** 
**What happened:**
Building ceph with ceph-ansible 5.0 stable (2020/11/04) 
Ansible-playbook deployment shows as successful.
Upon investigation the MDS status is stuck in "creating", under `ceph status`. `ceph fs status` also shows it as being `creating`.  
A 'crashed' container also appears on each controller node with a similar timestamp as the MDS container.
And there are items listed under `ceph osd blacklist ls` immediately 

**What you expected to happen:**
Ceph with cephfs in a working order

**How to reproduce it (minimal and precise):**
Latest ceph-ansible 5.0 stable (2020/11/04)
`ansible-playbook -v -i /opt/ceph-ansible/inventory -e 'ansible_python_interpreter=/usr/bin/python3' /opt/ceph-ansible/site-container.yml`

- Group_vars - http://sprunge.us/uUlS1N
- Inventory - http://sprunge.us/8JOSo7
- Ansible.log (-vv) - http://sprunge.us/C7pZim
- `ceph osd tree` - http://sprunge.us/brr1LX
- `ceph osd dump` - http://sprunge.us/5hpHEm
- `ceph pg dump` - http://sprunge.us/vEhRcj

```ceph osd blacklist ls
listed 5 entries
10.102.76.36:6800/54 2020-11-05T09:13:14.798350+0200
10.102.76.36:0/1704729219 2020-11-05T09:13:14.798350+0200
10.102.76.36:0/3111397685 2020-11-05T09:13:14.798350+0200
10.102.76.36:6801/54 2020-11-05T09:13:14.798350+0200
10.102.76.36:0/3819409455 2020-11-05T09:13:14.798350+0200
```
When I remove the blacklist items above & fail the MDS over to a new MDS, it immediately starts with new blacklist items.

```
 ceph osd blacklist ls
listed 2 entries
10.102.76.36:6803/2535027803 2020-11-05T09:42:09.750221+0200
10.102.76.36:6802/2535027803 2020-11-05T09:42:09.750221+0200
```

`ansible  -i /opt/ilifu_ceph-config/inventory osds,mons -m shell -a "ping 10.102.76.36 -c 2" -e 'ansible_python_interpreter=/usr/bin/python3'` - all nodes can ping that IP, no firewall between nodes in cluster. all nodes can ping each other from both network interface ranges

**Environment**:
* OS: `Ubuntu 18.04.5 LTS`
* Kernel `5.4.0-48-generic #52~18.04.1-Ubuntu SMP Thu Sep 10 12:50:22 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux`
* Docker version: `latest docker that comes with ceph-ansible` 
* Ansible version: `ansible-playbook 2.9.0`
* ceph-ansible version: `stable-5.0 (2020/11/04)`
* Ceph version (e.g. `ceph -v`): `15.2.5 `

**Comment by jeevadotnet** [comment]
TLDR; network misconfig


Problem: Our outsourced network resource changed something on the fabric side which caused it to work incorrectly. 
I just find it weird that ceph still deploys successful with no error.
